### PR TITLE
Fixes #10962:  Add ssl based authentication to policies

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/10962

### CODE REVIEW

File: techniques/system/distributePolicy/1.0/apache-acl.st [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#####################################################################################
+# Copyright 2014 Normation SAS
```
Comment (amousset) [inline techniques/system/distributePolicy/1.0/apache-acl.st]: 2017

File: techniques/system/distributePolicy/1.0/metadata.xml [hunk 2]
```diff
@@ -42,7 +53,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       <INCLUDED>false</INCLUDED>
     </TML>
     <TML name="nodeslist.json">
-      <OUTPATH>distributePolicy/1.0/nodeslist.json</OUTPATH>
```
Comment (amousset) [inline techniques/system/distributePolicy/1.0/metadata.xml]: is this the default value?

File: techniques/system/distributePolicy/1.0/apache-acl.st [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      "any" usebundle => rudder_common_report("DistributePolicy", "log_info", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache has been reloaded successfully");
+
+    pass3.apache_reload_failed::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache failed to reload");
+
+  commands:
+
+    rudder_apache_acl_repaired::
+
+      "${paths.path[service]}"
```
Comment (amousset) [inline techniques/system/distributePolicy/1.0/apache-acl.st]: Why not `service_reload("${apache_service}")`?

File: techniques/system/distributePolicy/1.0/metadata.xml (hunk 2 above)
Comment (peckpeck) [inline techniques/system/distributePolicy/1.0/metadata.xml]: yes

**Comment by peckpeck** [comment]
Commit modified

File: techniques/system/distributePolicy/1.0/apache-acl.st [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+    rudder_apache_acl_repaired.!rudder_apache_acl_error::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_repaired", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache ACLs have been repaired");
+
+    rudder_apache_acl_error::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache ACLs could not be edited");
+
+    rudder_apache_acl_repaired::
+      "any" usebundle => rudder_common_report("DistributePolicy", "log_info", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache has been reloaded successfully"),
+            ifvarclass => "${apache_service}_repaired"
```
Comment (amousset) [inline techniques/system/distributePolicy/1.0/apache-acl.st]: `service_reload_${apache_service}_repaired`?

**Comment by peckpeck** [comment]
Commit modified

File: techniques/system/distributePolicy/1.0/apache-acl.st [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_success", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache ACLs are correct");
+
+    rudder_apache_acl_repaired.!rudder_apache_acl_error::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_repaired", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache ACLs have been repaired");
+
+    rudder_apache_acl_error::
+      "any" usebundle => rudder_common_report("DistributePolicy", "result_error", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache ACLs could not be edited");
+
+    rudder_apache_acl_repaired::
+      "any" usebundle => rudder_common_report("DistributePolicy", "log_info", "&TRACKINGKEY&", "Configure apache ACL", "None", "Apache has been reloaded successfully"),
```
Comment (amousset) [inline techniques/system/distributePolicy/1.0/apache-acl.st]: log_repaired ?

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**